README.md

```diff
@@ -208,10 +208,31 @@ Accessing and modifying the default resource is done through two functions:
 rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource(); // Points to `cuda_memory_resource`
 // Construct a resource that uses a coalescing best-fit pool allocator
 rmm::mr::pool_memory_resource<rmm::mr::cuda_memory_resource>> pool_mr{mr}; 
-rmm::mr::set_default_resource(&pool_mr); // Updates the default resource pointer to `pool_mr`
+rmm::mr::set_current_device_resource(&pool_mr); // Updates the current device resource pointer to `pool_mr`
 rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource(); // Points to `pool_mr`
 ```
 
+#### Multiple Devices
+
+A `device_memory_resource` should only be used when the active CUDA device is the same device
```

Comment:
Technically this isn't true for resources like `cuda_memory_resource`. I think that is just an implementation detail though and not worth calling attention to. In general, the pattern should be one `device_memory_resource` object per device.